cmd/sops/common/common.go

```diff
@@ -143,10 +154,14 @@ func DefaultStoreForPath(path string) Store {
 	return &json.BinaryStore{}
 }
 
-const KMS_ENC_CTX_BUG_FIXED_VERSION = "3.3.0"
+// KmsEncCtxBugFixedVersion represents the SOPS version in which the
+// encryption context bug was fixed
+const KmsEncCtxBugFixedVersion = "3.3.0"
```

Comment:
This should stay as `KMS_ENC_CTX_BUG_FIXED_VERSION`

Reply:
✅